static/app/views/performance/transactionSummary/transactionEvents/content.tsx

```diff
@@ -0,0 +1,246 @@
+import * as React from 'react';
+import {Fragment} from 'react';
+import {browserHistory} from 'react-router';
+import styled from '@emotion/styled';
+import {Location, LocationDescriptor, Query} from 'history';
+import omit from 'lodash/omit';
+
+import SearchBar from 'app/components/events/searchBar';
+import * as Layout from 'app/components/layouts/thirds';
+import {getParams} from 'app/components/organizations/globalSelectionHeader/getParams';
+import Pagination from 'app/components/pagination';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {Organization, Project} from 'app/types';
+import DiscoverQuery, {TableDataRow} from 'app/utils/discover/discoverQuery';
+import EventView from 'app/utils/discover/eventView';
+import {generateEventSlug} from 'app/utils/discover/urls';
+import {decodeScalar} from 'app/utils/queryString';
+import {stringifyQueryObject, tokenizeSearch} from 'app/utils/tokenizeSearch';
+import {Actions, updateQuery} from 'app/views/eventsV2/table/cellAction';
+import {TableColumn} from 'app/views/eventsV2/table/types';
+import {getTraceDetailsUrl} from 'app/views/performance/traceDetails/utils';
+
+import {getCurrentLandingDisplay, LandingDisplayField} from '../../landing/utils';
+import {getTransactionDetailsUrl} from '../../utils';
+import {SpanOperationBreakdownFilter} from '../filter';
+import TransactionHeader, {Tab} from '../header';
+
+import TransactionsTable from './transactionsTable';
+
+const DEFAULT_TRANSACTION_LIMIT = 12;
+
+type Props = {
+  location: Location;
+  eventView: EventView;
+  transactionName: string;
+  organization: Organization;
+  isLoading: boolean;
+  totalValues: Record<string, number> | null;
+  projects: Project[];
+  spanOperationBreakdownFilter: SpanOperationBreakdownFilter;
+  cursorName: string;
+  limit: number;
+};
+
+class EventsPageContent extends React.Component<Props> {
```

Comment:
If there's a lot of duplication with the code in the transaction summary page, can this component be pulled into the transaction summary page with a different limit? Could be a follow up task

Reply:
Hey Shruthi, true there is a lot of duplicate code here, largely because the events tab replicates the functionality of the events table in the transaction summary page. I decided to break the transaction events stuff out into its own component despite the common code as I didn't want to impact the existing transaction summary page. At the moment, I think it would be feasible to do as you suggest, but I also think we may have interest in developing the transaction events tab in ways that we may not want reflected in the transaction summary. I remember the `errors` column was a potential addition that the team had discussed. I kept the code separate for this reason to try and avoid any future conflict. Do you think we should tackle this issue now rather than later?

Reply:
Yup, I think I'm fine with revisiting this if we feel like we are duplicating a lot of the Transaction Summary code. Maybe there can be a card for this task to see if it can be cleaned up once the feature is more complete. I just worry about duplicated code because now there are two places we'd need to update the same code but at the same time we don't want tightly coupled code which will be a pain to split up as new features get being added.